tests/test_gbdesAstrometricFit.py

```diff
@@ -409,15 +409,8 @@ def test_get_exposure_info(self):
                     tanPlaneXY[:, 0], tanPlaneXY[:, 1], degrees=True
                 )
 
-                # One WCS is in SIP and the other is TPV. The pixel-to-sky
-                # conversion is not exactly the same but should be close.
-                # TODO: sip_tpv + astropy.wcs.WCS gets a better result here,
-                # particularly for detector # >= 100. See if we can improve/if
-                # improving is necessary. Check if matching in corner detectors
-                # is ok.
-                rtol = 1e-3 if (detector >= 100) else 1e-5
-                np.testing.assert_allclose(calexpra, newRAdeg, rtol=rtol)
-                np.testing.assert_allclose(calexpdec, newDecdeg, rtol=rtol)
+                np.testing.assert_allclose(calexpra, newRAdeg)
```

Comment:
It's nice to be able to simplify this test so it no longer has to account for edge detectors.

Reply:
Yes, I was quite happy about that, and that the transformations for the other detectors are more accurate too.